### ISSUE TITLE
Flatpak: Options -> Font dialogues not responding (repost)

### ISSUE THREAD

**Issue by Alexey-T** [flathub/io.github.cudatext.CudaText-Qt#5]
See https://github.com/Alexey-T/CudaText/issues/5500

**Comment by JordanL2** [comment]
Interesting, if I run the flatpak's executable directly (bypassing the flatpak container), this issue does not occur:
```
LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HOME/.local/share/flatpak/app/io.github.cudatext.CudaText-Qt/current/active/files/lib $HOME/.local/share/flatpak/app/io.github.cudatext.CudaText-Qt/current/active/files/bin/cudatext
```
I have Qt6.7 in my host OS, while the flatpak currently uses 6.6. Unfortunately, we have to wait until the next stable release of Lazarus before it'll compile Qt6pas with Qt6.7.

**Comment by Alexey-T** [comment]
Thanks for checking..

**Comment by JordanL2** [comment]
The other possibility is the font dialogs are trying to access something on the system, and the flatpak container is preventing it from happening. Do you have any ideas what that could be? Perhaps:
1. something on the filesystem, eg /usr/lib or /etc
2. a session or system bus

In the meantine, I'll try enabling access to everything I can and see if that fixes it.

**Comment by JordanL2** [comment]
Enabling literally every option in Flatseal didn't fix the issue, so if it is a permission issue, seems like it won't be possible to fix right now.

**Comment by JordanL2** [comment]
I just testing running the executable directly with *only* the flatpak's lib in LD_LIBRARY_PATH, so in theory that'll only use the flatpak's libs and none of my host OS libs. The issue still did not occur, so I don't think it's an issue with any of the libraries the flatpak comes with (including Qt), but must be a permission issue of some kind.
```
LD_LIBRARY_PATH=/home/jordan/.local/share/flatpak/app/io.github.cudatext.CudaText-Qt/current/active/files/lib ~/.local/share/flatpak/app/io.github.cudatext.CudaText-Qt/current/active/files/bin/cudatext
```

**Comment by JordanL2** [comment]
This has been fixed by the update to Qt 6.8 and Lazarus 3.6.
